frontend/src/views/Room.jsx

```diff
@@ -324,6 +319,11 @@ function Room() {
     setIsSharingScreen(!isSharingScreen);
   };
 
+  const updateLocalTracksMuted = (kind, muted) => {
+    localTracks[kind].muted = muted;
+    setLocalTracks({ ...localTracks });
+  };
+
```

Comment:
Is this part of fixing the screenshare bug?

Reply:
No, this function was already there, it got moved to a different location.